app/assets/stylesheets/_search_result.scss

```diff
@@ -55,7 +55,11 @@
     .notes,
     .videos {
       float: left;
+      padding-top: 5px;
```

Comment:
now that the back button is never hidden does this extra padding still need to be here?